src/ui/Header.tsx

```diff
@@ -0,0 +1,6 @@
+import { Flex } from 'rebass'
+import styled from 'styled-components'
+
+export const Header = styled(Flex)`
+  border-bottom: 1px solid ${({ theme }) => theme.bg3};
+`
```

Comment:
Wondering what the idea behind this new ui folder? Seems a little out of place with just isolated files with one styled component in it...

Reply:
It's used in UserDashboard Liquidity and Loading pages. Was planning to move common UI components to one place.

Reply:
But these are styled components I think we were thinking more in direction of reusable components. These can go into some styled section once we start switching. And also we should adapt these to storybook...

Reply:
These are reused components. I moved it to a StyledElements folder in UI.